docs/developer-guide/syntax-reference.rst

```diff
@@ -1217,22 +1217,61 @@ Scalar functions
+|                        |                           |            | first considering all the records from the first window, then the   |
+|                        |                           |            | late-arriving record, then the records from the second window in    |
+|                        |                           |            | the order they were originally processed.                           |
++------------------------+---------------------------+------------+---------------------------------------------------------------------+
+| COLLECT_SET            | ``COLLECT_SET(col1)``     | Stream     | Return an array containing the distinct values of ``col1`` from     |
+|                        |                           |            | each input row (for the specified grouping and time window, if any).|
+|                        |                           |            | Currently only works for simple types (not Map, Array, or Struct).  |
+|                        |                           |            | This version limits the size of the result Array to a maximum of    |
+|                        |                           |            | 1000 entries and any values beyond this limit are silently ignored. |
+|                        |                           |            | When using with a window type of ``session``, it can sometimes      |
+|                        |                           |            | happen that two session windows get merged together into one when a |
+|                        |                           |            | late-arriving record with a timestamp between the two windows is    |
+|                        |                           |            | processed. In this case the 1000 record limit is calculated by      |
+|                        |                           |            | first considering all the records from the first window, then the   |
+|                        |                           |            | late-arriving record, then the records from the second window in    |
+|                        |                           |            | the order they were originally processed.                           |
++------------------------+---------------------------+------------+---------------------------------------------------------------------+
+| COUNT                  | ``COUNT(col1)``           | Stream,    | Count the number of rows                                            |
+|                        |                           | Table      |                                                                     |
++------------------------+---------------------------+------------+---------------------------------------------------------------------+
+| HISTOGRAM              | ``HISTOGRAM(col1)``       | Stream,    | Return a map containing the distinct values of ``col1`` mapped to   |
```

Comment:
This should note that it only works for string column types.

Reply:
done